experimental/BasisLieHighestWeight/src/MainAlgorithm.jl

```diff
@@ -1,8 +1,7 @@
 function basis_lie_highest_weight_compute(
-  L::LieAlgebra,
-  highest_weight::Vector{Int},
+  M::ModuleData,
```

Comment:
I think it would be more common to name the module `V` instead of `M` (everywhere)